src/system_tests/Test_mps.h

```diff
@@ -11,55 +11,62 @@
  **
  ** [[ Add lengthier description here ]]
 
- **/
+**/
+
+#include <cxxtest/TestSuite.h>
 
 #include "Engine.h"
 #include "InputQuery.h"
 #include "MpsParser.h"
-#include <cstdio>
+#include "MString.h"
 
-class Mps_Lp_Feasible_1
+class MpsTestSuite : public CxxTest::TestSuite
 {
 public:
-    void run()
+
+    void setUp()
+    {
+    }
+
+    void tearDown()
+    {
+    }
+
+    void test_infesiable() 
     {
-        const char *filename = "./lp_feasible_1.mps";
+        const String filename = RESOURCES_DIR  "/mps/lp_infeasible_1.mps";
 
         // Extract an input query from the network
         InputQuery inputQuery;
 
         MpsParser mpsParser( filename );
         mpsParser.generateQuery( inputQuery );
-
-        int outputStream = redirectOutputToFile( "logs/mps_lp_feasible_1.txt" );
-
-        struct timespec start = TimeUtils::sampleMicro();
-
         Engine engine;
         if ( !engine.processInputQuery( inputQuery ) )
         {
-            struct timespec end = TimeUtils::sampleMicro();
-            restoreOutputStream( outputStream );
-            printFailed( "mps_lp_feasible_1", start, end );
-            return;
+            // got infeasible in preprocess stage
+            TS_ASSERT( 1 );
```

Comment:
TS_ASSERT( 1 )? Maybe 0?

Reply:
It's infeasible test, I don't want to fail the test here, just wanted clarify that it's a good situation.
seems from the comment that I didn't do it well enough, any other idea how to say that this is a good situation (that the preprocessing failed)